api/v1alpha1/clusterextension_types.go

```diff
@@ -25,6 +25,15 @@ import (
 type UpgradeConstraintPolicy string
 
 const (
+
+	// TypeProgressing indicates whether operator-controller is
+	// reconciling, installing, updating or deleting an extension.
+	TypeProgressing = "Progressing"
+
+	ReasonProgressing                = "Progressing"
+	ReasonFailedToReachDesiredIntent = "FailedToReachDesiredIntent"
+	ReasonReachedDesiredIntent       = "ReachedDesiredIntent"
```

Comment:
_IIRC_ these were related to upgrades and being used in extension-controller. These can be removed too.